UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -42,7 +56,7 @@ public void BeforeTest(ITest test)
 
             Assert.AreEqual(ApartmentState.STA, Thread.CurrentThread.GetApartmentState());
 
-            // This form created for obtain UI synchronization context only
+            // This form is created to obtain an UI synchronization context only.
```

Comment:
```suggestion
            // This form is created to obtain a UI synchronization context only.
```

Reply:
> "a UI"

Really not "an UI"?

Reply:
Correct. The use of 'a' or 'an' is determined by the pronunciation, not the spelling. In this case, 'UI' is read out loud as 'you eye', with 'you' being a consonant sound that goes with 'a'.